### ISSUE TITLE
Only get 45% accuracy on UCF101 without pre-trained on kinetics?

### ISSUE THREAD

**Issue by Huixxi** [okankop/Efficient-3DCNNs#29]
Is that normal for 3d-mobilenetv2 1.0 width model batch_size 64 trained on UCF101 from scratch for 100 epoch without pre-trained on kinetics dataset can only get 45% accuracy?

**Comment by okankop** [comment]
Yes that is normal since there is overfitting. As a reference, ResNet-18 achieves around 47% clip accuracy when trained from scratch on UCF dataset.

**Comment by Huixxi** [comment]
> Yes that is normal since there is overfitting. As a reference, ResNet-18 achieves around 47% clip accuracy when trained from scratch on UCF dataset.

Very fast reply, thank you very much.
